DeliBuddy/app/src/main/java/yapp/android1/delibuddy/ui/PermissionViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package yapp.android1.delibuddy.ui
+
+import android.Manifest
+import android.os.Build
+import android.util.Log
+import androidx.lifecycle.ViewModel
+import com.gun0912.tedpermission.PermissionListener
+import com.gun0912.tedpermission.normal.TedPermission
+
+class PermissionViewModel : ViewModel() {
+    private val TAG = "PermissionViewModel"
+
+    val permissionListener: PermissionListener = object : PermissionListener {
+        override fun onPermissionGranted() {
+            Log.w(TAG, "Permission Granted!")
+        }
+
+        override fun onPermissionDenied(deniedPermissions: MutableList<String>?) {
+            Log.w(TAG, "Permission Denied!")
+        }
+    }
+
+    fun checkPermissions() {
+        if (Build.VERSION.SDK_INT >= 23) { // 마시멜로(안드로이드 6.0) 이상 권한 체크
```

Comment:
```suggestion
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
요렇게하면, 주석 안적어도 괜찮을듯 ㅎㅎ